oversweet-api/src/main/java/com/depromeet/oversweet/member/controller/MemberController.java

```diff
@@ -0,0 +1,73 @@
+package com.depromeet.oversweet.member.controller;
+
+import com.depromeet.oversweet.member.dto.KakaoTokenRequest;
+import com.depromeet.oversweet.member.dto.NicknameValidationResponse;
+import com.depromeet.oversweet.member.dto.SignUpRequest;
+import com.depromeet.oversweet.member.dto.SignUpResponse;
+import com.depromeet.oversweet.member.dto.SocialSignInResponse;
+import com.depromeet.oversweet.member.service.MemberFacade;
+import com.depromeet.oversweet.response.DataResponse;
+import com.depromeet.oversweet.util.JwtTokenProvider;
+import com.depromeet.oversweet.util.TokenResponse;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.servlet.http.Cookie;
+import jakarta.servlet.http.HttpServletResponse;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+
+@Slf4j
+@Tag(name = "회원", description = "회원 관련 API")
+@RestController
+@RequestMapping("/api/v1/members")
+@RequiredArgsConstructor
+public class MemberController {
+
+    private final MemberFacade memberFacade;
+    private final JwtTokenProvider jwtTokenProvider;
+
+    @Operation(summary = "닉네임 중복 체크", description = "닉네임 중복 체크 API 입니다.")
+    @ApiResponses({@ApiResponse(responseCode = "200", description = "닉네임 중복 체크 성공")})
+    @PostMapping("/validation")
```

Comment:
validation으로 하면 member의 어떤 부분을 validation 하는지 알 수 없어, 구체적인 상황을 표현할 수 있도록 하면 어떨까요?
```suggestion
    @PostMapping("/nickname-check")
```

Reply:
query parameter 로 닉네임을 받다보니 nickname 을 엔드포인트에 명시하지는 않긴 했는데, 이렇게 분명히 하는 것도 좋을 것 같긴합니다.
혹은 `validation/nickname` 은 어떤가요?

Reply:
좋습니다~